lib/rentlinx/client.rb

```diff
@@ -96,8 +96,7 @@ def response_handler(response)
       when 204
         nil # don't attempt to JSON parse emptystring
       when 400
-        body = JSON.parse(response.body)
-        raise Rentlinx::BadRequest, response, body['details']
+        raise Rentlinx::BadRequest, response
```

Comment:
So to be sure I understand: the functional change is to use the two-param version of `Kernel#raise` because the 3-param version is excepting an array?